modules/microshift-default-settings.adoc

```diff
@@ -75,9 +81,13 @@ node:
 <11> Default ports shown. Configurable. Valid values for both port entries are a single, unique port in the 1-65535 range. The values of the `ports.http` and `ports.https` fields cannot be the same.
 <12> Describes how hostname claims across namespaces are handled. By default, allows routes to claim different paths of the same hostname across namespaces. Valid values are `Strict` and `InterNamespaceAllowed`. Specifying `Strict` prevents routes in different namespaces from claiming the same hostname. If the value is deleted in a customized {microshift-short} `config.yaml`, the `InterNamespaceAllowed` value is automatically set.
 <13> Default router status, can be `Managed` or `Removed`.
-<14> The locations on the file system to scan for `kustomization` files to use to load manifests. Set to a list of paths to scan only those paths. Set to an empty list to disable loading manifests. The entries in the list can be glob patterns to match multiple subdirectories.
-<15> A block of IP addresses from which pod IP addresses are allocated. This field is immutable after installation.
-<16> A block of virtual IP addresses for Kubernetes services. IP address pool for services. A single entry is supported. This field is immutable after installation.
-<17> The port range allowed for Kubernetes services of type `NodePort`. If not specified, the default range of 30000-32767 is used. Services without a `NodePort` specified are automatically allocated one from this range. This parameter can be updated after the cluster is installed.
-<18> The name of the node. The default value is the hostname. If non-empty, this string is used to identify the node instead of the hostname.
-<19> The IP address of the node. The default value is the IP address of the default route.
+<14> Optional parameter for configuring low latency. Default value is null.
```

Comment:
```suggestion
<14> Optional parameter for configuring kubelet. Default value is null.
```
Technically someone could configure anything related to kubelet using that field. We don't really want that, but saying it's for "configuring low latency" feels off.